cluster/src/main/java/com/hartwig/pipeline/calling/sage/SageCaller.java

```diff
@@ -35,8 +35,8 @@
     public static final String SAGE_GENE_COVERAGE_TSV = "sage.gene.coverage.tsv";
     public static final String SAGE_BQR_PNG = "sage.bqr.png";
 
-    private final PersistedDataset persistedDataset;
-    private final SageConfiguration sageConfiguration;
+    protected final PersistedDataset persistedDataset;
```

Comment:
These protected fields are not accessed by the subclasses, so I think they can remain private